Quantum-Control-Applications-QuAM/Superconducting/quam_libs/components/transmon_pair.py

```diff
@@ -0,0 +1,24 @@
+from typing import Dict, Any, Optional, Union
+from dataclasses import field
+
+from quam.core import QuamComponent, quam_dataclass
+from .transmon import Transmon
+from .tunable_coupler import TunableCoupler
+
+
+__all__ = ["TransmonPair"]
+
+
+@quam_dataclass
+class TransmonPair(QuamComponent):
+    id: Union[int, str]
+    qubit_control: Transmon = None
+    qubit_target: Transmon = None
```

Comment:
We haven't reached concensus yet on whether to use qubit_control & qubit_target, or something else

Reply:
It's fine to merge, just so you know this may become a breaking change in the future

Reply:
Let's decide on the name then? control and target looks okay to me

Reply:
The root question is whether it makes sense in general for two-qubit gates to have a control and target. There are gates for which this doesn't make sense (e.g. symmetric gates such as CZ). I think it still makes sense to use `control` and `target`, but we need to reach alignment with the Arch team. Should be resolved soon, for now let's stick to this